### ISSUE TITLE
Query over the forward function of all models

### ISSUE THREAD

**Issue by ankitpatnala** [TUM-LMF/BreizhCrops#35]
https://github.com/dl4sits/BreizhCrops/blob/6de796ed36a457c8520322d6110b8f2862fd8c25/breizhcrops/models/PETransformerModel.py#L40-L52  

I wanted to implement your models on my data and I found at line 50 of  script ```PETransformerModel.py" log_softmax function has been implemented.

Secondly, when I saw your ```examples/train.py``` , I found criterion used is CrossEntropyLoss

https://github.com/dl4sits/BreizhCrops/blob/6de796ed36a457c8520322d6110b8f2862fd8c25/examples/train.py#L36-L40
https://github.com/dl4sits/BreizhCrops/blob/6de796ed36a457c8520322d6110b8f2862fd8c25/examples/train.py#L170-L182

So, the loss is calculated on log_softmax which is not recommended by pytorch as nn.CrossEntropy Loss function already does that in its subroutine. (https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html).
I found that it has been done in other models too.

I am not sure whether it was done intentionally or an implementation bug or a mistake in my understanding. Can you explain me before I run my computation.
